subiquity/server/controllers/tests/test_filesystem.py

```diff
@@ -127,3 +137,154 @@ def test_good_modes(self, mode):
     def test_bad_modes(self, mode):
         with self.assertRaises(ValueError):
             self.fsc.validate_layout_mode(mode)
+
+
+bootloaders = [(bl, ) for bl in list(Bootloader) if bl != Bootloader.NONE]
+
+
+class TestGuidedV2(IsolatedAsyncioTestCase):
```

Comment:
I realize it would be painful to write but shouldn't this test the POST method as well? It feels like we should post all the options back and check that the model things it's configured at least?

Reply:
I had a few already, so adding another isn't too painful.
See new commits.